report_dynamic/README.rst

```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
```

Comment:
to my knowledge the oca bot doesn't overwrite existing readmes, so better delete this or generate it from the current files